src/main/java/mate/academy/rickandmorty/controller/ApiController.java

```diff
@@ -0,0 +1,25 @@
+package mate.academy.rickandmorty.controller;
+
+import java.util.List;
+import lombok.RequiredArgsConstructor;
+import mate.academy.rickandmorty.model.CharacterEntity;
+import mate.academy.rickandmorty.service.impl.CharacterServiceImpl;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.RequestParam;
+import org.springframework.web.bind.annotation.RestController;
+
+@RestController
+@RequiredArgsConstructor
+public class ApiController {
+    private final CharacterServiceImpl characterService;
```

Comment:
use interface over implementation 
```suggestion
    private final CharacterService characterService;
```